### ISSUE TITLE
Figure out why # of Solr docs went down after index rebuild

### ISSUE THREAD

**Issue by mjgiarlo** [sul-dlss/rialto#68]

**Comment by jcoyne** [comment]
Can you offer some instructions for replicating the problem?

**Comment by mjgiarlo** [comment]
@jcoyne After loading sample data via SPARQL Update API on the dev server, I triggered an index rebuild via the Vitro admin UI. And there were 204,773 docs in the Solr index beforehand, and 123,498 afterward.

**Comment by jcoyne** [comment]
@mjgiarlo so is this blocked by #72?

**Comment by mjgiarlo** [comment]
@jcoyne No, we have two SPARQL Update-related load scenarios. The one related to this issue is #50 

I am inclined to close this issue, though, and keep an eye out for index weirdness as we test more of these scenarios.
